GetJobber/atlantis
# refactor(components): Update Styling of Tabs

## Motivations

With the Retheme we need to update the styling of Tabs to match our new designs

## Changes

### Changed

- Changed styling of Tabs to match retheme



## Testing

- Verify Tabs render correctly in the normal case and when in a modal
- Verify Tabs Overflow works in the normal case and in a modal.
- Use `git revert d6339fb1d2fcf12865680246b2e4e38527f01aaf` to get the modal story to use tabs
- Verify that this works in all browsers (Safari, Firefox and Chrome)


### After

Modal with overflow

<img width="660" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/81a10a6e-b3e9-4b4d-a6b7-60e2675dea30">

Modal no overflow
<img width="659" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/955246c8-d7ee-45e1-8b54-f2a02483888b">



Normal tabs
<img width="1275" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/86ab0319-7779-4bf3-ab2d-0b5c680fb643">

Normal Tabs with overflow
<img width="720" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/65a53c1d-141a-4bbe-af1e-4cd78d7da2d3">


---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
